packages/react-components/src/components/table-output-component.tsx

```diff
@@ -252,12 +272,16 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
     }
 
     private onModelUpdated = async () => {
-        if (this.pendingIndex && this.gridApi) {
-            if (this.gridApi.getSelectedNodes().length === 0) {
-                this.gridApi.getDisplayedRowAtIndex(this.pendingIndex).setSelected(true);
+        this.pendingIndices.forEach(index => {
+            if (this.gridApi) {
+                const node = this.gridApi.getDisplayedRowAtIndex(index);
+                if (node.id) {
+                    node.setSelected(true);
+                    this.gridApi.ensureIndexVisible(index);
```

Comment:
I don't think we need to ensureIndexVisible for an index that is not the start time. This means that it will show the last index and not the start index, right? Or do I miss something.